openpathsampling/pathmover.py

```diff
@@ -384,6 +384,8 @@ def metropolis(cls, trials):
         accepted = True
         probability = 1.0
 
+        # TODO: This isn't right. `bias` should be associated with the 
+        # change; not with each individual sample. ~~~DWHS
```

Comment:
This should not be a method. That's my point. It breaks the ability to maintain detailed balance for arbitrary moves. The overall acceptance is not of a sample. It is of an entire active sample set. In most moves, most of the sample set doesn't change, so this doesn't matter. However, the current structure of path movers will not work in the general case. See #251, #294.